examples/User Creation & Provisioning/README.md

```diff
@@ -33,19 +33,56 @@ This PowerShell script uses the Box CLI to build and create an onboarding folder
-    ```
-    firstName,lastName,email
-    Managed,User 1,ManagedUser1@test.com
-    Managed,User 2,ManagedUser2@test.com
-    Managed,User 3,ManagedUser3@test.com
-    ```
-3. Update the [RootFolderParentID](/examples/User%20Creation%20%26%20Provisioning/Users_Create_Provision.ps1#L21) to set destination folder ID for your changes, either when using JSON file as input to create folder structure, or uploading local structure. It is set to `0` by default, but feel free to set it to your needs.
-4. Update the [FolderStructureJSONPath](/examples/User%20Creation%20%26%20Provisioning/Users_Create_Provision.ps1#L13) to set your own Folder Structure JSON path. You can also change the [RootFolderName](/examples/User%20Creation%20%26%20Provisioning/Users_Create_Provision.ps1#L17). It's a name of the folder, that will be created as parent for folders from JSON structure. It's set to `Onboarding` by default, but feel free to set it to your needs.
-5. Optional: To directly upload folder structure from local directory, update [LocalUploadPath](/examples/User%20Creation%20%26%20Provisioning/Users_Create_Provision.ps1#L14), comment out [New-Folder-Structure](/examples/User%20Creation%20%26%20Provisioning/Users_Create_Provision.ps1#L113), and uncomment [RootFolderID](/examples/User%20Creation%20%26%20Provisioning/Users_Create_Provision.ps1#L116-L117). 
+## Configure the script
+### Update the list of employee accounts for creation
+For example, update the [employees_5.csv](/examples/User%20Creation%20%26%20Provisioning/Employees_5.csv) with the following data:
+```
+firstName,lastName,email
+Managed,User 1,ManagedUser1@test.com
+Managed,User 2,ManagedUser2@test.com
+Managed,User 3,ManagedUser3@test.com
+```
+### List of parameters
+
+- `EmployeeList`: Path to Employee List CSV.
+- `RootFolderParentID`: Destination folder ID for your changes, either when using JSON file as input to create folder structure, or uploading local structure. It is set to `0` by default, but feel free to set it to your needs.
+- `FolderStructureJSONPath`: Your own Folder Structure JSON Path. You can also change the `RootFolderName`. It's a name of the folder, that will be created as parent for folders from JSON structure. It's set to `Onboarding` by default, but feel free to set it to your needs.
+- `LocalUploadPath`: Local directory to directly upload folder structure.
+
+**Note**: Please specify either a local upload path or a folder structure JSON path, not both.
+
+### Define script parameters
+
+There are 3 ways to pass parameters:
+* Use hardcoded value in script:
+
+Please update all needed parameters in the script [here](/examples/User%20Creation%20%26%20Provisioning/Users_Create_Provision.ps1#L26-L38) before running.
+
+* Run script with parameters:
+
+You can also specify parameters while run the command, for example:
+```
+PS > ./Users_Create_Provision.ps1 -EmployeeList ./Employees_1.csv `
+	-LocalUploadPath ./OnboardingLocalUpload `
+	-RootFolderName Onboarding `
```

Comment:
Since the `RootFolderName` isn't required when doing Local upload, suggest to use json and pass `FolderStructureJSONPath` instead of `LocalUploadPath` flag. So we can showcase most of the params